data_steward/test/unit_test/resources_test.py

```diff
@@ -34,7 +34,7 @@ def test_cdm_schemas(self):
         self.assertListEqual([], result_achilles_tables,
                              msg='Achilles tables %s should not be in result of cdm_schemas()' % result_achilles_tables)
 
-        result_vocab_tables = [table_name for table_name in table_names if table_name in vocabulary.VOCABULARY_TABLES]
+        result_vocab_tables = [table_name for table_name in table_names if table_name in resources.VOCABULARY_TABLES]
```

Comment:
Since you have removed the dependence on the vocabulary module, you should remove the module import statement at the top of the file.

`import vocabulary`